dotcom-rendering/src/model/enhance-H2s.ts

```diff
@@ -78,10 +87,11 @@ const enhance = (elements: CAPIElement[]): CAPIElement[] => {
 	return enhanced;
 };
 
-export const enhanceH2s = (blocks: Block[]): Block[] =>
-	blocks.map((block: Block) => {
+export const enhanceH2s = (blocks: Block[]): Block[] => {
+	return blocks.map((block: Block) => {
 		return {
```

Comment:
Why wrapt this in a block?

Reply:
Why not wrap it in a block? 🙃

Reply:
Because it wasn’t wrapped in a block before!

Do you think we could take a stab at implementing https://eslint.org/docs/latest/rules/arrow-body-style for consistency?